stable/dex/templates/secret.yaml

```diff
@@ -49,4 +49,20 @@ stringData:
     {{- if .frontend }}
     frontend: {{ toYaml .frontend | nindent 6 }}
     {{- end }}
+    connectorFiltersHooks:
+{{ toYaml .connectorFiltersHooks | indent 6 }}
+    tokenClaimsHooks:
+{{ toYaml .tokenClaimsHooks | indent 6 }}
     {{- end }}
+    staticClients:
+    - id: example-app
+      redirectURIs:
+        - 'http://localhost:9009/auth/callback'
+      name: 'Example App'
+      secret: ZXhhbXBsZS1hcHAtc2VjcmV0
+    staticPasswords:
+      - email: "admin@example.com"
+        # bcrypt hash of the string "password": $(echo password | htpasswd -BinC 10 admin | cut -d: -f2)
+        hash: "$2a$10$2b2cU8CPhOTaGrs1HRQuAueS7JTT5ZHsHSzYiFPm1leZck7Mc8T4W"
+        username: "admin"
+        userID: "08a8684b-db88-4b73-90a9-3cd1661f5466"
```

Comment:
Do we need this section?

Reply:
No, thanks. Some leftovers from testing.

Reply:
030920e105693768c5a7cd40cfbe041732f64434